### PR TITLE
sys-fs/xwmfs: use sys-fs/fuse SLOT 0 in dependency

### PR DESCRIPTION
**Pull request by gerstner-hub**
Bug: https://bugs.gentoo.org/673632
Package-Manager: Portage-2.3.51, Repoman-2.3.11

### CODE REVIEW

File: sys-fs/xwmfs/xwmfs-0.82.ebuild [hunk 1]
```diff
@@ -12,7 +12,7 @@ KEYWORDS="~amd64 ~x86 ~arm"
 SLOT="0"
 
 RDEPEND="
-	sys-fs/fuse:=
+	sys-fs/fuse:0=
```
Comment (juippis) [inline sys-fs/xwmfs/xwmfs-0.82.ebuild]: I'm pretty sure this requires a revbump for everyone to register the change. Then, you can also update this ebuild to EAPI-7.

Also please add a sign-off as guided, info in https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin

**Comment by juippis** [comment]
With EAPI-7, `virtual/pkgconfig` belongs to BDEPEND. Also please don't use `git mv` but separate commits for revbump and removal of old ebuild. 

(To update sign-off status, edit the github issue title with `[please reassign]`)

**Comment by gerstner-hub** [comment]
> With EAPI-7, `virtual/pkgconfig` belongs to BDEPEND. Also please don't use `git mv` but separate commits for revbump and removal of old ebuild.
> 
> (To update sign-off status, edit the github issue title with `[please reassign]`)

Okay, should be covered. Sorry for the mistakes, I don't have much practice with ebuild maintenance.

File: sys-fs/xwmfs/xwmfs-0.82-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="FUSE-based X11 Window-Manager file system"
+HOMEPAGE="https://github.com/gerstner-hub/xwmfs"
+SRC_URI="https://github.com/gerstner-hub/${PN}/releases/download/v${PV}/${P}-dist.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86 ~arm"
```
Comment (juippis) [inline sys-fs/xwmfs/xwmfs-0.82-r1.ebuild]:
```
  KEYWORDS.unsorted             1
   sys-fs/xwmfs/xwmfs-0.82-r1.ebuild contains unsorted keywords
```
~arm before ~x86.

**Comment by juippis** [comment]
>Sorry for the mistakes, I don't have much practice with ebuild maintenance.

It's fine, you're doing good! I tried to emerge this but tests fail. I think you need to fix them, or restrict them. Can you test with `FEATURES="test" emerge -av xwmfs` to figure why it's failing?

Without tests it compiles fine.

**Comment by gerstner-hub** [comment]
Keywords are now sorted. I've unset `DISPLAY` to skip tests requiring a working X environment, this should fix the test feature.

**Pull request closed without merging**